definitions/ext-flex_host/summary_metrics.yml

```diff
@@ -0,0 +1,36 @@
+displayName:
```

Comment:
the name of the entity will be there without needing to add it here so I would remove this one.

Reply:
removed thanks